Use GitHub icon for source code link

simonbaird opened the pull request:
I thought it looked better on the right, so I also moved it.

simonbaird committed:
Use GitHub icon for source code link

I thought it looked better on the right, so I also moved it.

lcarva commented on antora/supplemental-ui/partials/header-content.hbs:
```diff
@@ -66,6 +64,13 @@
+        {{! GitHub logo SVG from https://icons.getbootstrap.com/icons/github/ }}
+        <a class="navbar-item" href="https://github.com/enterprise-contract" title="Source Code">
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor" class="bi bi-github" viewBox="0 0 16 16">
+            <path d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 ...
```
It would be nice if this came from a variable. But I have no idea if that's possible.

simonbaird replied:
It could be in its own partial probably..

simonbaird replied:
Pushed a new commit which uses a shared file with symlinks in two places. Maybe not the most beautiful solution, but it seems to work okay.

simonbaird committed:
Add a faster antora build Makefile target

When you're hacking there's no need to fetch source repos every
time, you can just use the cached version.

Not sure how to avoid the `make reference-docs` that runs in the
ec-cli repo, so that's still happening.

Co-authored-by: Zoran Regvart <zoran@regvart.com>

simonbaird committed:
Use shared file with symlinks for GitHub icon svg

The symlink technique seems like it has potential, but it's a little tricky
since there are two different templating languages being used. In
this case the shared file doesn't have any templating in it, so it
works fine.

(Could probably be squashed with previous commit, but I decided to
keep it separate to better show how the symlink was done.)

zregvart commented on antora/package.json:
```diff
@@ -17,7 +17,8 @@
   "scripts": {
-    "build": "antora generate --clean --fetch antora-playbook.yml"
+    "build": "antora generate --clean --fetch antora-playbook.yml",
+    "fast-build": "antora generate antora-playbook.yml"
```
Perhaps

```suggestion
    "build:base": "antora generate antora-playbook.yml",
    "build": "npm run build:base -- --clean --fetch",
    "build:fast": "npm run build:base",
```

simonbaird replied:
Done, thanks.

simonbaird commented:
Let's merge.
